### DISCUSSION TITLE
ScrollArea in Card component gives horizontal scrolling.

### DISCUSSION THREAD

**Discussion by melihcoban** [mantinedev/mantine#4589] [Q&A]
Giving ScrollArea as a component into Card gives me unwanted horizontal scroll in the bottom. I tried zeroing margin and padding the card but it does not work. Can I disable horizontal scrolling?

```
    <Card className={classes.card} component={ScrollArea} shadow="md" radius="md" withBorder>
      {children}
    </Card>
```
![image](https://github.com/mantinedev/mantine/assets/38351761/4058f614-616d-45d2-8ac4-f754419ae0f4)
![image](https://github.com/mantinedev/mantine/assets/38351761/b70ed874-d70a-487e-a5f6-17ff021b13c1)

**Comment by rtivital** [comment]
Render ScrollArea inside card instead of passing it to `component` prop

**Reply by melihcoban** [reply]
@rtivital Problem still continues.

![image](https://github.com/mantinedev/mantine/assets/38351761/3304fc39-06d9-41a8-9802-f4ca95f50448)

```
  const {height, width} = useViewportSize();

  const useStyles = createStyles(theme => ({
    card: {
      height: height - 130,
    },
  }));

  const {classes, theme} = useStyles();

  return (
    <Card className={classes.card} shadow="md" radius="md" withBorder>
      <ScrollArea.Autosize mah={height - 150} mx="auto">
        {children}
      </ScrollArea.Autosize>
    </Card>
  );
```

I tried without Autosize as well.

**Reply by rtivital** [reply]
Then your chridren exceed 100% width, make sure that they do not
